pom.xml

```diff
@@ -194,4 +194,10 @@
         </profile>
     </profiles>
 
+  <repositories>
+    <repository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
Out of curiosity, why?

Reply:
Try building without this hunk and you will see.